requirements-dev.txt

```
@@ -0,0 +1 @@
+pytest==*
```

You can just remove completely the `==*`
```suggestion
pytest
```